alf/optimizers/optimizers.py

```diff
@@ -188,6 +200,10 @@ def step(self, closure=None):
 
         super(NewCls, self).step(closure=closure)
 
+        for param in params:
+            if param.grad is not None and getattr(param, 'fixed_norm', False):
+                param.data.mul_(self._norms[param] / (param.norm() + 1e-30))
```

Comment:
It's not quite clear what the operation this ``fixed_norm` option represents. Maybe we can add some explanations.